build.rs

```diff
@@ -47,6 +47,8 @@ fn main() {
         })
         .collect();
 
+	tags.sort_by(|t1, t2| t1.id.cmp(&t2.id));
```

Comment:
why

Reply:
Because the tags appear in an os-dependent (I think?) ordering in the slash command popup

Reply:
well aren't we only running this on linux in prod? does it really matter anyways?

Reply:
It doesn't make any sense as it's not typically alphabetical and depends on filesystem (it's basically ls -U afaik)